tutorials/shadow_texture_size.md

```diff
@@ -0,0 +1,116 @@
+\page shadow_texture_size Shadow Texture Size
+
+This tutorial showcases how to change the texture size of shadows casted by lights.
+
+<div style="text-align:center;">
+  \image html files/shadow_texture_size/shadow_texsize_2k_to_16k.gif width=60%
+</div>
+
+## Texture size options
+
+Supported texture sizes include: 512 px, 1024 px, 2048 px, 4096 px, 8192 px, 16384 px.
+
+The default shadow texture size for all lights is 2K. The default max texture size is 16K, but for less powerful computers, it may be a lower value, like 8K.
+
+## Changing shadow texture size
+
+The shadow texture size can be changed for the GUI with the usage of the Minimal Scene GUI plugin in the SDF.
+
+### Example usage for the GUI
+
+Let's change the shadow texture size for directional light with the SDF file below. (The finished SDF file can be viewed [here](https://github.com/gazebosim/gz-sim/blob/main/examples/worlds/shadow_texture_size.sdf).)
```

Comment:
can you mention that we currently only support changing shadow texture size for directional lights?

Reply:
Yup = done here b48725b50d1b4fd554ec15e56d9c8e9d8d6c6823

Reply:
oh can you also change the link to point to gz-sim9 branch? i.e. https://github.com/gazebosim/gz-sim/tree/gz-sim9/examples/worlds/shadow_texture_size.sdf